templates/component/js/stats.js.twig

```diff
@@ -12,9 +12,6 @@
                     fontFamily: 'inherit',
                     height: 440,
                     parentHeightOffset: 0,
-                    toolbar: {
-                        show: false,
-                    },
```

Comment:
it can be useful
![Screenshot from 2020-05-07 09-22-45](https://user-images.githubusercontent.com/8239917/81266010-523b1100-9044-11ea-89c1-54ffa1659d74.png)